src/master_sword.c

```diff
@@ -0,0 +1,28 @@
+#include "master_sword.h"
+#include "kirby.h"
+#include "functions.h"
+
+static void sub_08025434(struct Object2* arg0);
+static void sub_08025440(struct Object2* arg0);
+
+struct Object2* CreateMasterSwordNonCollectible(struct Object* arg0, u8 arg1) {
```

Comment:
```suggestion
void* CreateMasterSwordNonCollectible(struct Object* arg0, u8 arg1) {
```
CreateXXX are placed in an array so we make all of them return void *.

Reply:
will do, thanks for the feedback